### ISSUE TITLE
Mixin members cannot have defaults

### ISSUE THREAD

**Issue by annevk** [KhronosGroup/WebGL#3540]
These appear on `WebGLRenderingContextBase`:
```webidl
     attribute PredefinedColorSpace drawingBufferColorSpace = "srgb";
     attribute PredefinedColorSpace unpackColorSpace = "srgb";
```
That's not correct IDL. Seems this was introduced in #3292.

Also, these attributes or `PredefinedColorSpace` don't appear in WebGL 2.0.

cc @kenrussell

**Comment by kdashg** [comment]
Fixed by https://github.com/KhronosGroup/WebGL/pull/3560.

**Comment by kenrussell** [comment]
Slight correction, was fixed by #3624.
